### ISSUE TITLE
Create a use case that explains the meta modelling requirement for domain/range restrictions on the dcat:contactPoint & other DCAT properties

### ISSUE THREAD

**Issue by andrea-perego** [w3c/dxwg#130]
> Action created during the [14 Feb 2018 DXWG DCAT subgroup call](https://www.w3.org/2018/02/14-dxwgdcat-minutes#x11)

Issues providing content for the use case:
* #95 (about `dcat:contactPoint`)
* #123 (about `dcat:theme`)

**Comment by dr-shorthair** [comment]
Related to #110

**Comment by jpullmann** [comment]
There is an initial UC draft [here](https://rawgit.com/w3c/dxwg/jaro_relax_property_domain_range_usecase/ucr/index.html#ID51), please review!

**Comment by andrea-perego** [comment]
Thanks, @jpullmann .

+1 from me, modulo a couple of fixes:

s/implying, that/implying that /

s/apporpiate/appropriate/

**Comment by dr-shorthair** [comment]
Thanks @jpullmann

**Comment by nicholascar** [comment]
I think addressing this Use Case as posed is impossible: we can't tackle domain & range restrictions for all properties all at once, *they are different*!

For the text as given so far:

s/weaking/weakening/
s/a property a part of guidance/a property, some of the guidance/
s/compensated by apporpiate/compensated for by appropriate/

Considering just the `dcat:contactPoint` domain & range:

* demoing the use of `dcat:contactPoint` with a relaxed domain:
    * The use of `dcat:contactPoint` for a catalogue is obviously appropriate and given that there are many circumstances within which people would want to contact the catalog maintainers rather than the maintainers of a specific dataset. The data.gov.au catalog (a CKAN instance able to export DCAT) has a non-semantically-described web page "About" with "Contact Us" information including the main catalogue team email address data@digital.gov.au. This should be able to be represented, legally, as follows:

```
:Catalog_x a dcat:Catalog ;
    dct:title "data.gov.au" ; # the catalog is called this!
    dcat:contactPoint <data@digital.gov.au>
.
```
* demoing the use of `dcat:contactPoint` with a relaxed range:
    * In the example above, yes a VCard could be used but perhaps it's onerous so a range relaxation might be sensible to use (pointing to just an eamil address as above).

**Comment by makxdekkers** [comment]
@nicholascar In terms of relaxing the range, what would you propose the range to be? Should it be something like `dcat:ContactInfo`, similar to  `schema:ContactPoint`? 
In the example you include, `dcat:ContactPoint <data@digital.gov.au>`, it seems like it's directly `rdfs:Literal` (like `schema:email`). Is that the intention?

**Comment by dr-shorthair** [comment]
I've re-written the proposed use-case to give us the lattitude to inspect all the global domain/range constraints. See [ID51 draft](https://rawgit.com/w3c/dxwg/simon-jaro_relax_property_domain_range_usecase/ucr/index.html#ID51)

**Comment by danbri** [comment]
FWIW (since schema.org equivalents were mentioned) if the WG would like term mappings (expressed in RDFS/OWL) included officially in Schema.org, that ought to be pretty straightforward. We have a few already. They are just additional assertions included in https://github.com/schemaorg/schemaorg/blob/master/data/schema.rdfa or nearby, and some of them are also reflected into RDFa/RDFS in per-term pages, e.g. view-source: on http://schema.org/Event shows an equivalency to dcterms:Event. As soon as we all agree what the mappings are, raise an issue at Schema.org's github and we'll get them reflected over there.

**Comment by andrea-perego** [comment]
This issue was addressed by #110 and linked issues.

I propose to close it.

**Comment by andrea-perego** [comment]
Closing it, as no objections have been raised since flagged as due for closing.
